### DISCUSSION TITLE
Lijst met BIO normen uit de NORA Information Security Object Repository voor server systemen

### DISCUSSION THREAD

**Discussion by klaver** [minvws/nl-kat-coordination#77] [Ideas]
Ik ben benieuwd of de BIO normen die NORA heeft verzameld in haar ISOR voor server systemen een handig uitgangspunt kunnen zijn voor het ontwikkelen van nieuwe boefjes. Een overzicht daarvan is hier te vinden: https://www.noraonline.nl/wiki/BIO_Thema_Serverplatform#Uitvoering_.28normen.29

Veel van die normen zijn procedurele zaken, maar een aantal zijn technisch van aard en zouden zich volgens mij prima lenen om middels een boefje geautomatiseerd te contoleren.

Enkele voorbeelden:

- SVP_U.09.03 heeft het onder andere over time-out faciliteiten, dit zou je bijvoorbeeld kunnen controleren op de aanwezigheid van specifieke configuratie parameters zoals ssh_timeout, of de TMOUT environment variable?
- SVP_U.05.02: contole of unattended-updates / yum-cron / etc... is ingericht?
- SVP_C.03.01: deze norm kan afgedekt worden door te controleren of auditd aanwezig is en op de juiste wijze is ingericht?

Zo zijn er vast nog wel meer checks te verzinnen!

**Comment by underdarknl** [comment]
Voor de procedurele zaken kunnen we ook controls inbouwen. 

SVP_U.01.03 Kent een aantal documenten die beschikbaar moeten zijn, bijvoorbeeld: bedieningsvoorschriften van de installatie en configuratie van systemen.
We kunnen natuurlijk prima een DocumentOOI introduceren waarmee je een document kunt invoegen als bewijs. Daarna kunnen we met de "SVP_U.01.03 BIT" controleren of en welke DocumentOOI's er in de graph beschikbaar zijn, en hoe oud deze bewijsstukken zijn. Afwezigheid levert dan een "SVP_U.01.03-BevindingOOI" op die daarmee het onmogelijk maakt om compliant te zijn op de BIO.
